### ISSUE TITLE
Import from PACS menu missing

### ISSUE THREAD

**Issue by Nunien** [fli-iam/shanoir-ng#348]
Seen by michael during tests
* Check if the import from PACS right is sync between shanoir db and keycloak after migration
* It could also have been a browser cache issue after redeploying

**Comment by michaelkain** [comment]
This is not a frontend bug, I think it comes from Keycloak. I started testing this morning,
my user mkain (expert), I checked with admin, has the right canImportFromPacs: yes using Manage users.
When I try to access to /imports/pacs, I get an answer, that I do not have the rights to do it.
Yesterday it worked, when we gave again the right to the user and communicated back to Keycloak,
this set the right in Keycloak, added it to the JWT and it works.

**Comment by michaelkain** [comment]
Hi Anthony, may I have access to Keycloak-qualif or could you verify, if the user mkain has the attribute canImportFromPacs=true. If not, we will have to figure out, why this attribute is not present in Keycloak.
This is an important bug, as Neurinfo imports from the pacs mostly and all three testers are experts.

**Comment by michaelkain** [comment]
I have analysed this. As Julien, I guess, was already thinking yesterday. We introduced the attribute canImportFromPacs later into Keycloak, what means the users in Keycloak (migrated by Anthony) do not have the flag yet, what is they do not have the flag on qualif. 10 users are concerned, as I do not want to waste time on automating this, we will tell the testers to manually reset the right, and I will set it once with the go live. So problem solved. If you have a better solution, let me know.
